docs/source/workflows/mcp/mcp-server.md

```diff
@@ -76,6 +76,44 @@ calculator_divide
 calculator_subtract
 ```
 
+### Debug route for listing tools (no MCP client required)
+You can also inspect the tools exposed by the MCP server without an MCP client by using the debug route:
+
+```bash
+curl -s http://localhost:9901/debug/tools/list | jq
+```
+
+This returns a JSON list of tools with names and descriptions.
+
+You can request one or more specific tools by name. The `name` parameter accepts repeated values or a comma‑separated list. When `name` is provided, detailed schemas are returned by default:
+
+```bash
```

Comment:
Can you add some sample outputs so we can see what format the outputs should be in?